src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -912,13 +921,40 @@ export default class DataTable extends Vue {
     this.slicedData = this.sliceData(this._sortedData || this._serializedDataBody);
   }
 
+  _showSelectedRows(isSelected: boolean) {
+    const selectedRow = this._serializedDataBody.filter((row: DataTableRow) => {
+      return this.selectedRows.includes(row.rowId);
+    });
+
+    for (const i in selectedRow) {
+      selectedRow[i].selected = true;
+    }
```

Comment:
```suggestion
```
Is this necessary?
`selected` property must be taken care of the moment the user selects it. I don't think there is a need of doing it here.

Reply:
Yes, as there is no way to get selected true value once checked, this is needed here.